src/sparc/client/services/pennsieve.py

```diff
@@ -0,0 +1,68 @@
+import abc
+import configparser
+import logging
+import os
+
+from pennsieve2 import Pennsieve
+from ._default import ServiceBase
+
+
+class PennsieveService(ServiceBase):
+
+    default_headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json; charset=utf-8",
+    }
+
+    host_api = "https://api.pennsieve.io"
+
+    def __init__(self, target="localhost:9000", config=None, connect=False) -> None:
```

Comment:
is ``target`` necessary here?

Reply:
Good catch! No, it's not. Pennsieve() class adds this as a default.